USF-2278: initial draft of changes for removing references to gdrive and sharepoint (#277)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/launch/index.mdx

1. [Fastly Setup](https://www.aem.live/docs/byo-cdn-fastly-setup): This guide illustrates how to configure Fastly to deliver content.
1. [CloudFront Setup](https://www.aem.live/docs/byo-cdn-cloudfront-setup): Set up Amazon Web Services Cloudfront to deliver your AEM site with push invalidation.
1. [Bring your own DNS](https://www.aem.live/docs/byo-dns): A custom domain without having to set up a content delivery network.

> 1. [Redirects](https://www.aem.live/docs/redirects): You can intuitively manage redirects as a spreadsheet called redirects (or `redirects` document) in the root of your project folder.

Reviewer: ```suggestion
1. [Redirects](https://www.aem.live/docs/redirects): You can manage redirects as a spreadsheet from the `redirects` document in the root of your project folder.
```

* [ ] Sync the production catalog to the new production environment.
* [ ] Create a new Commerce production API key-pair and use the public key as the `x-api-key` value.
* [ ] Double check category IDs and make sure all category pages have a reference to the correct category.

> * [ ] Update `environmentId` and `x-api-key` values in the site public config or `config.json` code file.

Reviewer: Remove public config for now.

Reviewer 2: Oops. missed this one. Fixing.
